recipe/meta.yaml

```diff
@@ -165,10 +164,6 @@ requirements:
     - xcb-util-keysyms                   # [linux]
     - xcb-util-image                     # [linux]
     - xcb-util-renderutil                # [linux]
-    # https://github.com/conda-forge/xorg-libx11-feedstock/pull/28
-    - xorg-libx11                        # [linux]
-    # https://github.com/conda-forge/xorg-libxext-feedstock/pull/12
-    - xorg-libxext                       # [linux]
```

Comment:
These run deps are removed because the mentioned pull requests have been merged.